docs/DeveloperGuide.md

```diff
@@ -699,7 +698,7 @@ unless specified otherwise)
 2.  Should be able to hold up to 1000 students without a noticeable sluggishness in performance for typical usage.
 3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
 
-- Business/domain rules: Each student must be uniquely identifiable by their full name, Assignments must have deadlines that  cannot be set on a date that has passed, Each class should not exceed 50 students, Assignments must be submitted by students up to the deadline set by the Teacher
+- Business/domain rules: Assignments must have deadlines that  cannot be set on a date that has passed, Assignments must be submitted by students up to the deadline set by the Teacher
```

Comment:
Was wondering if we still need to include deadlines in the rules since our product doesnt have it? 

Maybe for business/domain rules some of it could be:
- Assignments cannot have maxScore of 0
- Student and assignment name cannot have have special characters in its name
- The submission status of a graded assignment cannot be 'Not Submitted'

These are just some potential examples, could add more also!